apps/admin/backend/src/exports/csv_tally_report.ts

```diff
@@ -22,16 +24,28 @@ import {
   generateCsvMetadataHeaders,
   getCsvMetadataRowValues,
 } from './csv_shared';
+import { tabulateManualResults } from '../tabulation/manual_results';
+
+// eslint-disable-next-line vx/gts-no-return-type-only-generics
+function assertIsOptional<T>(_value?: unknown): asserts _value is Optional<T> {
+  // noop
+}
 
 function generateHeaders({
   election,
   metadataStructure,
+  hasManualResults,
 }: {
   election: Election;
   metadataStructure: CsvMetadataStructure;
+  hasManualResults: boolean;
 }): string[] {
   const headers = generateCsvMetadataHeaders({ election, metadataStructure });
-  headers.push('Contest', 'Contest ID', 'Selection', 'Selection ID', 'Votes');
+  headers.push('Contest', 'Contest ID', 'Selection', 'Selection ID');
+  if (hasManualResults) {
+    headers.push('Manual Votes', 'Scanned Votes');
+  }
+  headers.push('Total Votes');
```

Comment:
If there's no manual results, we could just keep the column name here as "Votes." But I assume for external systems consuming our .csv files, it will be less confusing if the column names do not change.